borg/archive.py

```diff
@@ -786,11 +791,11 @@ def add_callback(chunk):
 
         def add_reference(id_, size, csize, cdata=None):
             try:
-                count, _, _ = self.chunks[id_]
-                self.chunks[id_] = count + 1, size, csize
```

Comment:
@RonnyPfannschmidt of course it only really uses cdata when repair mode is engaged. but calling this function for a chunk that is not in the repo and not providing the cdata (even though not needed in "dry-run not do repair mode") can be considered an error also (as it would fail when trying to do the repair).